### PR TITLE
qasm parser docstring + code simplification

### PR DESCRIPTION
**Pull request by balopat**
Nits after #1718.

### CODE REVIEW

File: cirq/contrib/qasm_import/_parser.py [hunk 1]
```diff
@@ -71,21 +82,23 @@ def on(self, args: List[List[cirq.Qid]],
...
 
 
 class QasmParser:
+    """QasmParser is the parser for QASM strings.
+
+        Example:
+
+         >>> qasm = "OPENQASM 2.0; qreg q1[2]; CX q1[0], q1[1];"
+         >>> parsedQasm = QasmParser().parse(qasm)
+         ```
```
Comment (vtomole) [inline cirq/contrib/qasm_import/_parser.py]:
```suggestion
         
```

**Comment by balopat** [comment]
@vtomole I added your suggestions as commits, please confirm your consent to use them here for the cla :)

**Comment by vtomole** [comment]
Confirming doesn't work in this case unfortunately. Craig will override during the merge.

File: cirq/contrib/qasm_import/_parser.py [hunk 2]
```diff
@@ -37,13 +39,28 @@ def __init__(self, supported_format: bool, qelib1_include: bool,
...
+        Has the responsibility to validate the arguments
+        and parameters of the call and to generate a list of corresponding
+        `cirq.GateOperation`s in the `on` method.
+        """
 
     def __init__(self, qasm_gate: str, cirq_gate: cirq.Gate, num_args: int):
         self.qasm_gate = qasm_gate
         self.cirq_gate = cirq_gate
         self.num_args = num_args
+        """Initializes a Qasm gate statement.
```
Comment (Strilanc) [inline cirq/contrib/qasm_import/_parser.py]: Docstring goes after def line

File: cirq/contrib/qasm_import/_parser.py [hunk 3]
```diff
@@ -71,21 +82,23 @@ def on(self, args: List[List[cirq.Qid]],
...
+                                    " at line {}".format(lineno))
             yield self.cirq_gate.on(*qubits)
 
 
 class QasmParser:
+    """Parser for QASM strings.
+
+        Example:
+
+         >>> qasm = "OPENQASM 2.0; qreg q1[2]; CX q1[0], q1[1];"
```
Comment (Strilanc) [inline cirq/contrib/qasm_import/_parser.py]: Don't indent `Example:`. Don't include the `>>>` before code; just indent it 4 spaces.

File: cirq/contrib/qasm_import/_parser.py (hunk 2 above)
Comment (balopat) [reply cirq/contrib/qasm_import/_parser.py]: god - that's why I shouldn't commit/push while on a meeting :)

File: cirq/contrib/qasm_import/_parser.py (hunk 3 above)
Comment (balopat) [reply cirq/contrib/qasm_import/_parser.py]: ack

**Comment by balopat** [comment]
@Strilanc this needs a manual override of the `cla:yes` label

**Comment by dabacon** [comment]
cla override reason: @vtomole changes incorporated and @vtomole has signed CLA

**Pull request merged**